_data/team.yml

```diff
@@ -91,14 +84,3 @@
       - bioinformatics
       - excercise
       - microbiome
-
-- name: Max von Hippel
```

Comment:
I decided to revoke this comment (just forgot to delete it in the review) since it doesn't really matter which ordering we pick here -- it's equally unclear which part of Karen's last name should be sorted on.